templates/build.and.release.invokebuild.yml

```diff
@@ -0,0 +1,183 @@
+parameters:
+  preCustomEnvironmentVariables: []
+  postCustomEnvironmentVariables: []
+  preBuild: []
+  postBuild: []
+  preCopyNugetPackages: []
+  postCopyNugetPackages: []
+  prePublishReleaseArtifacts: []
+  postPublishReleaseArtifacts: []
+  preCreateGitHubRelease: []
+  postCreateGitHubRelease: []
+  prePublishNugetPackages: []
+  postPublishNugetPackages: []
+  vmImage: ''
+  service_connection_nuget_org: '' 
+  service_connection_github: '' 
+  build_script: ''
+  build_task: '.'
+  netSdkVersion: '3.x'
+
+jobs:
+- job: Build
+  pool:
+    vmImage: ${{ parameters.vmImage }}
+
+  variables:
+    BuildConfiguration: 'Release'
+    DOTNET_CLI_TELEMETRY_OPTOUT: 1
+    DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
+
+    # We have dependencies on the following Environment Variables:
+    # GITVERSION_PRERELEASETAG
```

Comment:
...but this Environment Variable only gets set as a result of running `install-and-run-gitversion.yml`. The `build.and.release.yml` template does this on line 53:

https://github.com/endjin/Endjin.RecommendedPractices.AzureDevopsPipelines.GitHub/blob/cc88c35874074b71cc3cbed4c1bdf58f0a566762/templates/build.and.release.yml#L53

But this modified InvokeBuild-based version appears not to call `install-and-run-gitversion.yml` so the `GITVERSION_PRERELEASETAG` variable will never actually get set, will it?

And in fact, we seem to rely on a bunch more outputs from Gitversion—it's not just the `GITVERSION_PRERELEASETAG` environment variable. There are various references to `$GitVersion.*` throughout this template.